static/js/utils.js

```diff
@@ -29,7 +29,7 @@ const calcTextWidth = (text) => {
 const propertyName = (property) => {
     if (columnsReady) {
         try {
-            return window.columns["keys"][property]["name"]
+            return window.columns.filter(c => c["key"] === property)[0]["name"]
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/ranimepiracy/index/pullRequest?prid=7256353)
```suggestion
            return window.columns.filter((c) => c["key"] === property)[0]["name"]
```